pkg/driverbuilder/builder/oracle.go

```diff
@@ -0,0 +1,107 @@
+package builder
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+
+	"github.com/falcosecurity/driverkit/pkg/kernelrelease"
+)
+
+//go:embed templates/oracle.sh
+var oracleTemplate string
+
+// TargetTypeoracle identifies the oracle target.
+const TargetTypeoracle Type = "oracle"
```

Comment:
Can we name it `ol` (that is ID from `/etc/os-release` in oraclelinux systems)?
```
✓ (457ms) 12:09:28 [federico][fededp-sysdig][update/oracle][~/Work/kernel-crawler] $ docker run -ti --rm oraclelinux:6 cat /etc/os-release | grep ID | head -n1
ID="ol"
✓ (576ms) 12:09:33 [federico][fededp-sysdig][update/oracle][~/Work/kernel-crawler] $ docker run -ti --rm oraclelinux:7 cat /etc/os-release | grep ID | head -n1
ID="ol"
✓ (617ms) 12:09:39 [federico][fededp-sysdig][update/oracle][~/Work/kernel-crawler] $ docker run -ti --rm oraclelinux:8 cat /etc/os-release | grep ID | head -n1
ID="ol"
✓ (586ms) 12:09:43 [federico][fededp-sysdig][update/oracle][~/Work/kernel-crawler] $ docker run -ti --rm oraclelinux:9 cat /etc/os-release | grep ID | head -n1
ID="ol"
```

Reply:
See also https://github.com/falcosecurity/kernel-crawler/pull/107

Reply:
Done!